Source/UI/NewUI/PaymentForm/View/PaymentHeaderView.swift

```diff
@@ -11,7 +11,11 @@ import UIKit
 public final class PaymentHeaderView: UIView {
   private var style: PaymentHeaderCellStyle?
 
-  private var stackView: UIStackView = {
+  private lazy var backgroundView: UIView = {
+    UIView().disabledAutoresizingIntoConstraints()
+  }()
```

Comment:
Do we need this "background view"? Looks like a complicated way to let the owner of the view to set it up as it wants.

Have you tried to just add paddings around the headerView inside PaymentViewController?

Reply:
Yes, we need it for header view.